docs-src/genomes.rst

```diff
@@ -74,3 +75,39 @@ Annotations
    The `.dganno` file contains the compiled ``GFF3`` and the `.cfg` file
    contains metadata, such as ``refg=hg38``.
 
+
+Appris / MANE
+-------------
+
+When adding an annotation, you can also generate Appris/MANE data files for it if public data is available.
+
+Appris files are available on https://apprisws.bioinfo.cnio.es/pub/releases/.
+As noted in the `GenomeKit source code <https://github.com/deepgenomics/GenomeKit/blob/41c1fbe5c011d04504b08eef4435c50771c01471/genome_kit/_build_appris.py#L107>`_,
+we maintain a partial archive of Appris.
+
+Gencode
+^^^^^^^
+
+For Gencode annotations, first find the matching Ensembl release. For example, for Gencode v47, the matching Ensembl
+release is 113. So you'll need to find an Appris release that includes e113. The earliest release that includes e113 is
```

Comment:
maybe link to https://www.gencodegenes.org/human/releases.html
so people can find the correspondence